src/components/NavigationBar/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Navbar, NavbarBrand, NavbarContent, NavbarItem, Link, Button } from "@nextui-org/react";
+import "./style.css";
+import { useNavigate } from "react-router-dom";
+
+export const NavigationBar = ({isLoggedin}) => {
+  const navigateTo = useNavigate();
+  
+  return (
+    (isLoggedin)?
+    <Navbar>
```

Comment:
This can be optimised to reduce your lines of code by adding the check only to the children of `NavbarContent`